packages/web-components/src/tree-item/tree-item.template.ts

```diff
@@ -0,0 +1,48 @@
+import { children, elements, html } from '@microsoft/fast-element';
+import type { TreeItem } from './tree-item.js';
+
+// We don't put the icon into the icons/index.ts file because
+// this icon is the default chevron icon of the tree-item component,
+// it would be contributed to the upstream later.
+const chevronIcon = html`
+  <svg width="12" height="12" xmlns="http://www.w3.org/2000/svg" fill="currentColor">
+    <path
+      d="M4.65 2.15a.5.5 0 000 .7L7.79 6 4.65 9.15a.5.5 0 10.7.7l3.5-3.5a.5.5 0 000-.7l-3.5-3.5a.5.5 0 00-.7 0z"
+    ></path>
+  </svg>
+`;
+
+export const template = html<TreeItem>`
+  <template
+    tabindex="-1"
+    slot="${x => (x.isNestedItem ? 'item' : void 0)}"
+    @focusin="${(x, c) => x.handleFocus(c.event as FocusEvent)}"
+    @focusout="${(x, c) => x.handleBlur(c.event as FocusEvent)}"
+    ${children({
+      property: 'childTreeItems',
+      filter: elements(),
+    })}
+  >
+    <div class="positioning-region" part="positioning-region">
+      <div class="content-region" part="content-region">
+        <span class="chevron-region" part="chevron-region">
+          <slot name="chevron">${chevronIcon}</slot>
+        </span>
+        <span class="start-region" part="start-region">
+          <slot name="start"></slot>
+          <slot></slot>
+          <slot name="end"></slot>
+        </span>
+      </div>
+      <div class="badging-region" part="badging-region">
+        <slot name="badging"></slot>
+      </div>
+      <div class="toolbar-region" part="toolbar-region">
```

Comment:
nit: the parts and classes being suffixed with `-region` feels redundant. Can these be reduced?

* content-region -> content
* chevron-region -> chevron
* start-region -> start
* badging-region -> badges
* toolbar-region -> toolbar